about/package-scope.md

```diff
@@ -195,6 +195,27 @@ Packages to aid with instruction.
 
 ## Package technical scope
 
+
+### Telemetry & user-informed consent
+
+Your package should avoid collecting usage analytics. With
```

Comment:
from @choldgraf @NickleDave maybe something like this instead for the first sentence?

Your package must avoid collecting usage analytics without informing people using it.